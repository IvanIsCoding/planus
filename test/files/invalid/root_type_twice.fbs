// We do not currently use root types for anything... :(
root_type X;
root_type Y;