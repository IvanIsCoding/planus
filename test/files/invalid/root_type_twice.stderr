[0m[1m[38;5;9merror[0m[1m: Cannot set root_type twice[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/root_type_twice.fbs:2:1
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m [0m[31mroot_type X;[0m
  [0m[34mâ”‚[0m [0m[31m^^^^^^^^^^^^[0m [0m[31mfirst declaration was here[0m
[0m[34m3[0m [0m[34mâ”‚[0m [0m[31mroot_type Y;[0m
  [0m[34mâ”‚[0m [0m[31m^^^^^^^^^^^^[0m [0m[31madditional declaration was here[0m

Error: Bailing because of errors
